src/workspace-names.cpp

```diff
@@ -284,25 +285,22 @@ class wayfire_workspace_names_output : public wf::per_output_plugin_instance_t
         auto wsn     = workspaces[x][y]->workspace;
         int ws_num   = x + y * wsize.width + 1;
 
+        // Get intended name of the workspace
```

Comment:
```suggestion
        // Get default name of the workspace
```

Reply:
Right!! This is not the name, this is the key. I'll correct it..